float64 targetLat 
float64 targetLon
float32 targetAlt
float32 targetRadius
uint32 loiterTime
float32 maxHorizSpeed 
float32 maxVertSpeed 
float64 poiLat 
float64 poiLon 
float32 poiHeading 
float32 poiAltitude 
uint32 flags
uint8 photo 
uint8 panoSectorsCount 
float32 panoDeltaAngle
float32 poiPitch
float32 poiRoll
uint8 type
